support-frontend/assets/pages/supporter-plus-landing/setup/threeTierChecks.ts

```diff
@@ -6,6 +6,17 @@ export const threeTierCheckoutEnabled = (
 	abParticipations: Participations,
 	countryId: IsoCountry,
 ): boolean => {
+	const isPaperCheckout = window.location.pathname.startsWith(
+		'/subscribe/paper/checkout',
+	);
+	const isDigitalEditionCheckout = window.location.pathname.startsWith(
+		'/subscribe/digitaledition/checkout',
```

Comment:
should this read `/subscribe/digitaledition` to match -> https://support.theguardian.com/uk/subscribe/digitaledition

Reply:
Good spot! You're correct, will update now.